docs/components/connectors/react-components/_connectors-table.js

```diff
@@ -0,0 +1,519 @@
+    },
+    {
+      name: "UiPath",
+      description: "Orchestrate your UiPath Bots with Camunda.",
+      type: "Outbound",
+      link: "../uipath/",
+      image: UipathImg,
+    },
+    {
+      name: "WhatsApp",
+      description: "Send messages with WhatsApp Business from your processes.",
+      type: "Outbound",
+      link: "../whatsapp/",
+      image: WhatsappImg,
+    },
+  ];
+
+  const handleSearch = (event) => {
+    setSearchTerm(event.target.value);
+  };
+
+  const handleFilterType = (type) => {
+    setFilterType(type);
+  };
+
+  const filteredConnectors = connectors.filter((connector) => {
+    const matchesSearchTerm =
+      connector.name.toLowerCase().includes(searchTerm.toLowerCase()) ||
+      connector.description.toLowerCase().includes(searchTerm.toLowerCase());
+    const matchesFilterType =
+      filterType === "All" || connector.type === filterType;
+    return matchesSearchTerm && matchesFilterType;
+  });
+
+  return (
+    <div className="connector-input">
+      <input
+        type="text"
+        placeholder="Find a Camunda connector"
+        value={searchTerm}
+        onChange={handleSearch}
+        className="connector-input-box"
+      />
+      <div className="filter-badges">
+        <span
+          className={`badge--default ${
+            filterType === "All" ? "badge--active" : ""
+          }`}
+          onClick={() => handleFilterType("All")}
+        >
+          All
+        </span>
+        <span
+          className={`badge--default ${
+            filterType === "Inbound" ? "badge--active--inbound" : ""
+          }`}
+          onClick={() => handleFilterType("Inbound")}
+        >
+          Inbound
+        </span>
+        <span
+          className={`badge--default ${
+            filterType === "Outbound" ? "badge--active--outbound" : ""
+          }`}
+          onClick={() => handleFilterType("Outbound")}
+        >
+          Outbound
+        </span>
+      </div>
+      <span className="connector-input-link-about">
+        <a href="../../connector-types">About connector types</a>
+      </span>
+      {filteredConnectors.length > 0 ? (
+        <div className="connector-grid">
+          {filteredConnectors.map((connector, index) => (
+            <a key={index} href={connector.link} className="connector-card">
+              <div className="connector-card-image">
+                <img src={connector.image} alt={connector.name} />
+              </div>
+              <div className="connector-name">
+                <h3>{connector.name}</h3>
+              </div>
+              <div className="connector-description">
+                <p>{connector.description}</p>
+              </div>
+              <div className="connector-type">
+                <span
+                  className={`badge badge--${connector.type.toLowerCase()}`}
+                >
+                  {connector.type}
+                </span>
+              </div>
+            </a>
```

Comment:
non-blocking: visually, the only difference I can see between the cards rendered on the introduction page vs the grid results is the presence of a badge. I would have expected to see the extracted Card component able to conditionally render a badge if it's provided, so that the card itself is only defined in one place.